.github/workflows/tests.yml

```diff
@@ -1,6 +1,17 @@
-name: CI
+name: Multi-software test
+
+
+
+on:
+  pull_request:
+    branches:
+    - master
+  push:
+    branches: 
+    - master
+    - develop
```

Comment:
this will only run tests in the master and developer branches and in pull requests that goes to master, right? what about a pull request to develop?

honestly i think we should test it in the pull request, no matter the destination, that is the point that matters the most, in my opinion, if something fails you are still in time to fix it and most of all it blocks the PR of being merged (it should btw, we probably have to update the settings in the github)